utils/social-tips.ts

```diff
@@ -0,0 +1,182 @@
+
+function mergeCoins(tipAmount: Coin[]): Coin[] {
+  let coins: Record<string, number> = {};
+
+  tipAmount.forEach(coin => {
+    const value = coins[coin.denom];
+    if (value === undefined) {
+      coins[coin.denom] = parseInt(coin.amount);
+    } else {
+      coins[coin.denom] = value + parseInt(coin.amount);
+    }
+  })
+
+  return Object.keys(coins).map(k => {
+    return {denom: k, amount: coins[k].toString()} as Coin
+  })
+}
+
+function setupExecuteCommand(command: Command, client: DesmosClient, account: AccountData) {
+  command.command("init")
+    .description("initialize a new instance of a social-tips contract")
+    .requiredOption("--code-id <code-id>", "id of the contract to initialize", parseInt)
+    .requiredOption("--max-pending-tips <max-pending-tips>", "id of subspace which is deploying the contract", parseInt)
+    .requiredOption("--max-sent-pending-tips <max-sent-pending-tips>", "the number of records saved of a user tips history", parseInt)
+    .requiredOption("--name <name>", "contract name")
+    .option("--admin <admin>", "bech32 address of who will have the admin rights", account!.address)
+    .action(async (options) => {
+      let instantiateMsg: InstantiateMsg = {
+        max_pending_tips: options.maxPendingTips,
+        max_sent_pending_tips: options.maxSentPendingTips,
+        admin: options.admin
+      };
+      // @ts-ignore
+      const initResult = await client.instantiate(account!.address, options.codeId, instantiateMsg, options.name, "auto");
+      console.log("Contract initialized", initResult);
+    })
+
+  command.command("tip-user")
+    .description("send a tip to an user")
+    .argument("<application>", "application name")
+    .argument("<handle>", "user handle in the provided application")
+    .requiredOption("--coins <coins>", "amount of coins to tip to the user. Comma separated list of coins ex: 100udsm,100uatom", parseCoinList)
+    .requiredOption("--contract <contract>", "bech32 encoded contract address")
+    .option("--owner-index <owner-index>", "index of the address to which the tip will be sent in case the user have connected the application to multiple profiles")
+    .action(async (application, handle, options) => {
+      const funds = mergeCoins([...options.coins]);
+      console.log("Tip amount", options.coins);
+
+      let contractClient = new SocialTipsClient(client, account!.address, options.contract);
+      let response = await contractClient.sendTip({
+        application,
+        handle,
+        ownerIndex: options.ownerIndex
+      }, "auto", undefined, funds);
+      console.log(response)
+    });
+
+
+  command.command("claim-tips")
+    .description("claim the user pending tips")
+    .requiredOption("--contract <contract>", "bech32 encoded contract address")
+    .action(async (options) => {
+      let contractClient = new SocialTipsClient(client, account!.address, options.contract);
+      let response = await contractClient.claimTips("auto");
```

Comment:
```suggestion
      let response = await contractClient.claimTips();
```
"auto" can be removed since the default value is "auto".